dev-system/dev/user.clj

```diff
@@ -199,7 +199,7 @@
   ;; Prevent jobs from blocking calls to reset
   (humanizer-report-service/set-retry-count! 0)
   (humanizer-report-service/set-humanizer-report-generator-job-wait! 0)
-  (content-service/set-generation-interval! 60)
+  (content-service/set-static-content-generation-interval! 60)
 
   ;; uncomment this line to start gorilla repl.
```

Comment:
Is this still needed?

Reply:
Yeah, it's good to have this in the dev env for testing new content changes.